<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-12T18:38:18 IST" name="Default suite" finished-at="2025-07-12T18:38:29 IST" duration-ms="11450">
    <groups>
    </groups>
    <test started-at="2025-07-12T18:38:18 IST" name="Default test" finished-at="2025-07-12T18:38:29 IST" duration-ms="11450">
      <class name="testCases.CabBookingPageTest">
        <test-method is-config="true" signature="DriverSetup()[pri:0, instance:testCases.CabBookingPageTest@477b4cdf]" started-at="2025-07-12T18:38:18 IST" name="DriverSetup" finished-at="2025-07-12T18:38:23 IST" duration-ms="4954" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DriverSetup -->
        <test-method is-config="true" signature="goToHomePage()[pri:0, instance:testCases.CabBookingPageTest@477b4cdf]" started-at="2025-07-12T18:38:23 IST" name="goToHomePage" finished-at="2025-07-12T18:38:28 IST" duration-ms="4903" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToHomePage -->
        <test-method signature="verifyMessageForUnserviceableToLocation(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.CabBookingPageTest@477b4cdf]" started-at="2025-07-12T18:38:28 IST" name="verifyMessageForUnserviceableToLocation" data-provider="testData" finished-at="2025-07-12T18:38:29 IST" duration-ms="1532" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Cabs]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Outstation One-Way]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delhi]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Manali, Himachal Pradesh]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Dec 09 2025]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[12:40 PM]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[SUV]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Mumbai] but found [No Data Found]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Mumbai] but found [No Data Found]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at testCases.CabBookingPageTest.verifyMessageForUnserviceableToLocation(CabBookingPageTest.java:218)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyMessageForUnserviceableToLocation -->
      </class> <!-- testCases.CabBookingPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
